tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -415,7 +415,7 @@ _ when x.ToBoolean() is { } boolean => boolean,
 
             DataPipelineEnabled = config
                                   .WithKeys(ConfigurationKeys.TraceDataPipelineEnabled)
-                                  .AsBool(defaultValue: true);
+                                  .AsBool(defaultValue: false);
```

Comment:
Yikes, we haven't done a release with it enabled by default have we?

Reply:
I'm just building more PRs on top of big ones to keep things small, concise and easy to review.